src/InjectionContainer.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\View;
+
+use Psr\Container\ContainerInterface;
+use RuntimeException;
+
+final class InjectionContainer implements InjectionContainerInterface
+{
+    public function __construct(
+        private ContainerInterface $container,
+    ) {
+    }
+
+    public function get(string $id): object
+    {
+        $result = $this->container->get($id);
+        return is_object($result)
```

Comment:
Is the check required? PHP will throw an exception if the type of result does not match to the return type.

Reply:
Agree. Done.